pkg/apis/core/v1beta1/kubefedcluster_types_test.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Do we need this test here? It doesn't compile and seems unnecessary. Delete please.

Reply:
I think we need to add more tests in general, I fixed the issues with this test. It works now.

Reply:
OK I'm not sure of the value of this test but I guess a test is better than nothing :)